boards/arm/nucleo_wb55rg/Kconfig.defconfig

```diff
@@ -42,9 +42,6 @@ config I2C_1
 config I2C_3
 	default y
 
-config I2C_STM32_INTERRUPT
-	default y
-
```

Comment:
Nice to get rid of these. :ok_hand: 

When it's an option that makes sense to toggle in the `menuconfig`, and you're just setting the default setting, then it's better to do it in configuration files.

Reply:
Unless the dependencies might not be satisfied. `Kconfig.defconfig` is better then, to avoid that assignment-didn't-take warning.

That part is a bit messy.